AcadeMe/urls.py

```diff
@@ -23,7 +23,7 @@
     path('admin/', admin.site.urls),
     path('', views.homePage,  name='homepage'),
     path('register/', user_views.register, name='register'),
-    path('login/', auth_views.LoginView.as_view(template_name='users/login.html'), name='login'),
+    path('login/', auth_views.LoginView.as_view(template_name='../templates/login/login.html'), name='login'),
```

Comment:
Made this change in order to use the html login page I created instead of @DanielItkes 's old one that was used to check the db post methods